chains/scripts/README.md

```diff
@@ -0,0 +1,3 @@
+usage: scripts to be run in the context of the repl
```

Comment:
I think we already have an option for this in the repl, if I recall it should be `yarn repl -s ./scripts/add_user.js`

Reply:
oh cool. is there one that keeps the shell open after running the script?